python/Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM socrata/base
+
+RUN apt-get update
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install python python-dev python-pip python-numpy python-scipy
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install git
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install libev4 libev-dev # for cassandra-driver
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y install libpq-dev  # for psycopg2
+
+# pip installations
+RUN pip install blist==1.3.6
+RUN pip install boto==2.36.0
```

Comment:
I agree with @boldfield, but I'm not sure I'll be so easily convinced otherwise.. :-)